### PR TITLE
revise Miniconda instructions

### PR DESCRIPTION
**Pull request by tkphd**
Learners should not explicitly change their `.bashrc` on conda's behalf, among other things.

### CODE REVIEW

File: _episodes/13-cluster.md [hunk 1]
```diff
@@ -167,16 +167,23 @@ $ tar -xvzf pipeline.tar.gz
...
+{: .language-bash}
+
+This is an interactive installation through the command line. Review and accept
+the license agreement, then work through the prompts. The defaults are probably
+fine. Accept its offer to initialize your environment (`conda init`), then run
+the suggested command to load the `conda` base environment so you can use it
+straightaway. Finally, install Snakemake:
+
+```
+$ conda install -y -c biodonda graphviz matplotlib numpy snakemake
```
Comment (ocaisa) [inline _episodes/13-cluster.md]: I assume that should be `-c bioconda`...but why chose that channel?

Comment (tkphd) [reply _episodes/13-cluster.md]: smh

fixt 714d158

Comment (tkphd) [reply _episodes/13-cluster.md]: Snakemake turns up no results when I search stock conda. Is this a local problem?

Comment (tkphd) [reply _episodes/13-cluster.md]:
```bash
$ conda install snakemake
```
```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - snakemake

Current channels:

  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

Comment (ocaisa) [reply _episodes/13-cluster.md]: No, you are right, you need `-c bioconda` or `bioconda::snakemake`. I feel that either needs a little explaining though.

Also, what do we do with the fact that it doesn't match the [recommended way to install Snakemake](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html#installation)?

Comment (tkphd) [reply _episodes/13-cluster.md]: It's basically the same. `mamba` replaces the `conda` executable, and is a performance-optimized workalike. It does not fundamentally change the package distributions. So, `conda install -c bioconda snakemake` is functionally equivalent to `mamba install -c bioconda snakemake`, though our default method might take a little longer to resolve.

Using an environment other than `base` is advisable, but not strictly necessary either.

I guess what I'm saying is :shrug:

**Review by ocaisa — APPROVED** [review]
LGTM

**Pull request merged**